twsi_upper_bound.py

```diff
@@ -30,10 +30,11 @@ def evaluate_uppper_bound(twsi_dataset_fath, user2twsi):
 def main():
     parser = argparse.ArgumentParser(description='Estimation of the upper bound performance given the custom Word Sense Inventory.')
     parser.add_argument('user_inventory', metavar='sense-inventory', help='word sense inventory file, format:\n word_senseID <tab> list,of,words')
+    parser.add_argument('predictions', help='word sense disambiguation predictions in the 9 column lexical sample format.')
     args = parser.parse_args()
 
     user2twsi = map_sense_inventories(TWSI_INVENTORY, args.user_inventory)
-    correct, count = evaluate_uppper_bound(TWSI_DATASET, user2twsi)
+    correct, count = evaluate_uppper_bound(args.predictions, user2twsi)
```

Comment:
can you please add to this line of code a default parameters equals to TWSI_DATASET?